Extension/lib/webrequest.js

```diff
@@ -178,9 +183,30 @@
             ruleForCSP = adguard.webRequestService.getRuleForRequest(tab, 'stun:adguardwebrtc.check', frameUrl, adguard.RequestTypes.WEBRTC);
             applyCSP = adguard.webRequestService.isRequestBlockedByRule(ruleForCSP);
         }
-
+        if (applyCSP) {

```

Comment:
Hm, I don't think so. These rules like some workaround, so they shouldn't be whitelisted by $csp rules. @ameshkov What do you think?